phase3/weave-net/weave-net-daemon-set.json

```diff
@@ -1,133 +1,163 @@
-                    },
-                    {
-                        "name": "dbus",
-                        "hostPath": {
-                            "path": "/var/lib/dbus"
-                        }
-                    },
-                    {
-                        "name": "lib-modules",
-                        "hostPath": {
-                            "path": "/lib/modules"
-                        }
-                    }
-                ]
-            }
-        }
-    }
-}
+	"apiVersion": "extensions/v1beta1",
+	"kind": "DaemonSet",
+	"metadata": {
+		"name": "weave-net",
+		"labels": {
+			"name": "weave-net"
+		},
+		"namespace": "kube-system"
+	},
+	"spec": {
+		"template": {
+			"metadata": {
+				"annotations": {
+					"scheduler.alpha.kubernetes.io/tolerations": "[{\"key\":\"dedicated\",\"operator\":\"Equal\",\"value\":\"master\",\"effect\":\"NoSchedule\"}]"
+				},
+				"labels": {
+					"name": "weave-net"
+				}
+			},
+			"spec": {
+				"hostNetwork": true,
+				"hostPID": true,
+				"containers": [
+					{
+						"name": "weave",
+						"image": "weaveworks/weave-kube:2.0.5",
+						"command": [
+							"/home/weave/launch.sh"
+						],
+						"env": [
+							{
+								"name": "HOSTNAME",
+								"valueFrom": {
+									"fieldRef": {
+										"apiVersion": "v1",
+										"fieldPath": "spec.nodeName"
+									}
+								}
+							}
+						],
+						"livenessProbe": {
+							"initialDelaySeconds": 30,
+							"httpGet": {
+								"host": "127.0.0.1",
+								"path": "/status",
+								"port": 6784
+							}
+						},
+						"securityContext": {
+							"privileged": true
+						},
+						"volumeMounts": [
+							{
+								"name": "weavedb",
+								"mountPath": "/weavedb"
+							},
+							{
+								"name": "cni-bin",
+								"mountPath": "/host/opt"
+							},
+							{
+								"name": "cni-bin2",
+								"mountPath": "/host/home"
+							},
+							{
+								"name": "cni-conf",
+								"mountPath": "/host/etc"
+							},
+							{
+								"name": "dbus",
+								"mountPath": "/host/var/lib/dbus"
+							},
+							{
+								"name": "lib-modules",
+								"mountPath": "/lib/modules"
+							}
+						],
+						"resources": {
+							"requests": {
+								"cpu": "10m"
+							}
+						}
+					},
+					{
+						"name": "weave-npc",
+						"image": "weaveworks/weave-npc:2.0.5",
+						"env": [
+							{
+								"name": "HOSTNAME",
+								"valueFrom": {
+									"fieldRef": {
+										"apiVersion": "v1",
+										"fieldPath": "spec.nodeName"
+									}
+								}
+							}
+						],
+						"resources": {
+							"requests": {
+								"cpu": "10m"
+							}
+						},
+						"securityContext": {
+							"privileged": true
+						}
+					}
+				],
+				"restartPolicy": "Always",
+				"tolerations": [
+					{
+						"key": "node-role.kubernetes.io/master",
+						"effect": "NoSchedule"
+					}
+				],
+				"serviceAccountName": "weave-net",
+				"securityContext": {
```

Comment:
I _think_ this was removed by WW some time ago

Reply:
this commit is just reformatting ... i am not adding or removing anything here